pages/api/create-event.js

```diff
@@ -34,7 +35,7 @@ async function storeFiles(files) {
 async function makeFileObjects(body) {
   const buffer = Buffer.from(JSON.stringify(body));
 
-  const imageDirectory = resolve(process.cwd(), "public/images/event.png");
+  const imageDirectory = resolve(process.cwd(), `public/images/${body.image}`);
```

Comment:
the subgraph currently looks for event.png ([reference](https://github.com/womenbuildweb3/web3RSVP-subgraph/blob/main/src/mapping.ts#L25)). it won't know what image file to look for since we're not storing that info  in the smart contract.

we should either 
a) rename the image file to event.png when uploading to web3 storage
or b) include the image file name in the metadata that is stored on web3.storage and then update the subgraph to read the image file name from the metadata